src/core/io/src/4C_io_input_spec_builders.hpp

```diff
@@ -295,6 +295,10 @@ namespace Core::IO
    */
   namespace InputSpecBuilders
   {
+    // Import the Noneable type into the InputSpecBuilders namespace to make it easier to use.
+    using Core::IO::none;
+    using Core::IO::Noneable;
```

Comment:
We We have been reluctant with unscoped `using` statements in the past. This one is not unscoped, though also not tightly encapsulated and part of a header file, so it might be included somewhere else.

I do not object in this case. Just wanted to build awareness...

Reply:
The whole intention is that you include the header, write `using namespace Core::IO::InputSpecBuilders;` and thus also gain access to the `Noneable` and  `none` symbols. See https://godbolt.org/z/4qYe4T951